.github/workflows/format.yml

```diff
@@ -54,8 +46,8 @@ jobs:
           submodules: true
       - name: 'Run DrString'
         env:
-          DEVELOPER_DIR: /Applications/Xcode_14.1.app
-        run: ./bazelw run @DrString//:drstring check
+          DEVELOPER_DIR: /Applications/Xcode_14.0.app
```

Comment:
Why did we revert from Xcode 14.1 to Xcode 14.0 here?

Reply:
https://github.com/envoyproxy/envoy/pull/24223

Reply:
argh,  my original pull was before the bump and the . files kept being problematic.  will fix thanks for the catch